aspnet/tutorials/web-api-help-pages-using-swagger.rst

```diff
@@ -0,0 +1,536 @@
+.. _web-api-help-pages-using-swagger:
+
+ASP.NET Web API Help Pages using Swagger
+========================================
+
+By `Shayne Boyer`_
+
+Understanding the various methods of an API can be a challenge for a developer when building a consuming application. 
+
+Generating good documentation and help pages as a part of your Web API using `Swagger <a href="http://swagger.io/">`_ with the .NET Core implementation `Swashbuckle <a href="https://github.com/domaindrivendev/Ahoy">`_ is as easy as adding a couple of nuget packages and modifying the Startup.cs.
+
+This tutorial builds on the sample on :doc:`first-web-api`. 
```

Comment:
Add
If you'd like to follow along, download the sample at https://github.com/aspnet/Docs/tree/master/aspnet/tutorials/first-web-api/sample.